homeassistant/components/bang_olufsen/media_player.py

```diff
@@ -299,19 +402,20 @@ async def _async_update_sources(self) -> None:
         # Combine the source dicts
         self._sources = self._audio_sources | self._video_sources
 
-        # HASS won't necessarily be running the first time this method is run
-        if self.hass.is_running:
```

Comment:
How come this check is now removed?

Reply:
Yeah, that shouldn't have been removed. I'll add it again.